apps/pipelines/models.py

```diff
@@ -227,6 +227,7 @@ def get_langchain_messages_until_summary(self):
 
 class PipelineChatMessages(BaseModel):
     chat_history = models.ForeignKey(PipelineChatHistory, on_delete=models.CASCADE, related_name="messages")
+    node_id = models.TextField()
```

Comment:
So this node id's purpose is purely informational?

Reply:
Currently, yes.